src/qt/coincontroldialog.cpp

```diff
@@ -229,9 +229,10 @@ void CoinControlDialog::copyAddress()
 }
 
 // context menu action: copy transaction id
```

Comment:
I guess you missed updating the comment.

```suggestion
// context menu action: copy transaction id and output index
```